### DISCUSSION TITLE
problema com ionic start

### DISCUSSION THREAD

**Discussion by matheusvarela** [balta-io/7188#27] [Perguntas e Respostas] [answered]
após a execução do comando ionic start eshop blank, é criado um projeto angular na versão 13.2.2
Deixei as versões do node, angular e ionic igual do video, mas o ionic insiste (rsrsr) colocar versão mais recente 

É possível especificar a versão do angula no comando ionic start?

parece que visual studio está ignorando a instalação -g (global) do npm

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @matheusvarela, no caso, teria que instalar a mesma versão do @ionic/angular para obter a versão que usamos... Pelo que sei ele não deixa alterar...

**Reply by matheusvarela** [reply]
![image](https://user-images.githubusercontent.com/11960349/166586422-7870cc66-8028-4451-8d99-1b269348360d.png)
![image](https://user-images.githubusercontent.com/11960349/166586811-bdfe4279-f1e5-4c79-9139-84ea20d79011.png)
Ao verificar a versão na pasta do projeto gerado
![image](https://user-images.githubusercontent.com/11960349/166586940-48986d1d-227c-4932-998a-eb8f92bb8db1.png)
![image](https://user-images.githubusercontent.com/11960349/166587107-ba253ebb-6824-4e0a-8c6c-f6593d370ccf.png)

Se executar o comando ng --version fora da pasta do projeto ele mostra a versão correta, mas na pasta do projeto ela usa mais rescente

![image](https://user-images.githubusercontent.com/11960349/166587452-57b803ff-e461-4881-93a2-0d9bf35428cd.png)

**Reply by andrebaltieri** [reply]
Bom dia, @matheusvarela, você pode obter as mesmas versões do curso no arquivo `packages.json` na raiz do repositório...
